source/common/router/router_ratelimit.h

```diff
@@ -98,7 +98,7 @@ class RemoteAddressAction : public RateLimitAction {
 class GenericKeyAction : public RateLimitAction {
 public:
   GenericKeyAction(const envoy::config::route::v3::RateLimit::Action::GenericKey& action)
-      : descriptor_value_(action.descriptor_value()) {}
+      : descriptor_value_(action.descriptor_value()), descriptor_key_(action.descriptor_key()) {}
```

Comment:
perf nit: you can do the ternary here, and avoid it on population above.

Reply:
Done